SourceX/controls/controller.cpp

```diff
@@ -22,9 +110,12 @@ ControllerButtonEvent ToControllerButtonEvent(const SDL_Event &event)
 	}
 
 #if HAS_KBCTRL == 1
-	result.button = KbCtrlToControllerButton(event);
-	if (result.button != ControllerButton_NONE)
-		return result;
+	KeyboardController *const keyboardController = KeyboardController::Get(event);

```

Comment:
Why not use a reference?

Reply:
This can be null, references cannot be null

Reply:
Ah yeah... makes sense, I forgot about that.